frontend-wasm/src/miden_abi/transform.rs

```diff
@@ -15,20 +19,40 @@ enum TransformStrategy {
 }
 
 /// Get the transformation strategy for a function name
-fn get_transform_strategy(function_id: &str) -> TransformStrategy {
-    match function_id {
-        tx_kernel::note::GET_INPUTS => TransformStrategy::ListReturn,
-        tx_kernel::account::ADD_ASSET => TransformStrategy::ReturnViaPointer,
-        tx_kernel::account::REMOVE_ASSET => TransformStrategy::ReturnViaPointer,
-        tx_kernel::account::GET_ID => TransformStrategy::NoTransform,
-        tx_kernel::tx::CREATE_NOTE => TransformStrategy::NoTransform,
-        stdlib::crypto::hashes::BLAKE3_HASH_1TO1 => TransformStrategy::ReturnViaPointer,
-        stdlib::crypto::hashes::BLAKE3_HASH_2TO1 => TransformStrategy::ReturnViaPointer,
-        stdlib::crypto::dsa::RPO_FALCON512_VERIFY => TransformStrategy::NoTransform,
-        stdlib::mem::PIPE_WORDS_TO_MEMORY => TransformStrategy::ReturnViaPointer,
-        stdlib::mem::PIPE_DOUBLE_WORDS_TO_MEMORY => TransformStrategy::ReturnViaPointer,
-        _ => panic!("No transform strategy found for function {}", function_id),
+fn get_transform_strategy(module_id: &str, function_id: &str) -> TransformStrategy {
+    #[allow(clippy::single_match)]
+    match module_id {
+        "std::mem" => match function_id {
+            stdlib::mem::PIPE_WORDS_TO_MEMORY => return TransformStrategy::ReturnViaPointer,
+            stdlib::mem::PIPE_DOUBLE_WORDS_TO_MEMORY => return TransformStrategy::ReturnViaPointer,
+            _ => (),
+        },
+        "std::crypto::hashes::blake3" => match function_id {
```

Comment:
`"std::crypto::hashes::blake3"` string literal is also used in `stdlib::crypto::hashes` module. It'd be better to define in one place.